src/server/worldserver/worldserver.conf.dist

```diff
@@ -1062,6 +1062,36 @@ Guild.EventLogRecordsCount = 100
 
 CacheDataQueries = 1
 
+#
+#   WowPatch
+#        Description: Set patch
+#        Default:     15 - (Patch 2.4.3)
+#                     0 = 1.2
+#                     1 = 1.3
+#                     2 = 1.4
+#                     3 = 1.5
+#                     4 = 1.6
+#                     5 = 1.7
+#                     6 = 1.8
+#                     7 = 1.9
+#                     8 = 1.10
+#                     9 = 1.11
+#                     10 = 1.12
+#                     11 = 2.0
+#                     12 = 2.1
+#                     13 = 2.2
+#                     14 = 2.3
+#                     15 = 2.4.3
+#                     16 = 3.0
+#                     17 = 3.1.0
+#                     18 = 3.2.0
+#                     19 = 3.2.2
+#                     20 = 3.3.0
+#                     21 = 3.3.5
+#
```

Comment:
Why start from patch 1.2 when this is a TBC+ emulator? It seems entirely pointless. Would make more sense if 0 was 2.0, 1 was 2.1, etc.

Reply:
I just included everything, because thats what i did when i ported this for tc.

Reply:
I agree with brotalnia on this as well